File: StelMap/CB/CB_Test/Unit1.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Unit1.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "stelmap"
#pragma resource "*.dfm"
TForm1 *Form1;
//---------------------------------------------------------------------------
__fastcall TForm1::TForm1(TComponent* Owner)
	: TForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TForm1::FormResize(TObject *Sender)
{
  StelMap1->MapResize();
  StelMap1->Prepare();
  StelMap1->LoadMap(0, 0, TRUE);
  StelMap1->ReDraw();
}
//---------------------------------------------------------------------------

File: StelMap/Unit1.pas
unit Unit1;

interface

uses
  Windows, Messages, SysUtils, Variants, Classes, Graphics,
  Controls, Forms, Dialogs, ExtCtrls, stelmap;

type
  TForm1 = class(TForm)
    StelMap1: TStelMap;
    procedure FormResize(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  Form1: TForm1;

implementation

{$R *.dfm}

procedure TForm1.FormResize(Sender: TObject);
begin
  StelMap1.MapResize;
  StelMap1.Prepare;
  StelMap1.LoadMap(0, 0, TRUE);
  StelMap1.ReDraw;
end;

end.
